.circleci/run-tests.sh

```diff
@@ -74,14 +74,14 @@ else
     alternative="false"
 fi
 
-${TIMEOUT} ${BOOTSTRAP_VENV}/bin/tox \
-    -c ${PROJECT_ROOT}/tox.ini \
+${TIMEOUT} "${BOOTSTRAP_VENV}"/bin/tox \
+    -c "${PROJECT_ROOT}"/tox.ini \
     --workdir /tmp/magic-folder.tox \
     -e "${MAGIC_FOLDER_TOX_ENVIRONMENT}" \
-    ${MAGIC_FOLDER_TOX_ARGS} || "${alternative}"
+    "${MAGIC_FOLDER_TOX_ARGS}" || "${alternative}"
```

Comment:
This quoting change is the only one I'm concerned about here.  I think that the point was to be able to pass multiple positional arguments to tox here - perhaps mostly for pass-through.  For example, `["--", "--dry-run", "magic_folder.tests.cli"]`.  I think the added quoting turns these all into a single argument in tox's argv and I don't know what the consequences of that are.

Maybe it's fine because the single string then expands to multiple strings when tox uses it to construct a shell command?  Because I don't think there's consistent quoting inside tox.ini for handling these...  But maybe there should be?

Is there a better way to handle multiple arguments in shell than this?  Should it be a shell array instead?